tests/e2e/labelupdates.go

```diff
@@ -783,3 +780,17 @@ func waitForPodNameLabelRemoval(volumeID string, podname string, namespace strin
 	}
 	return fmt.Errorf("pod name label is not removed from cns")
 }
+
+func waitForPVCDeletion(volumeHandle string, pvcName string, namespace string) error {
```

Comment:
leave a comment above function name, to document what it is doing
also func name is not matching its actions